File: Java/Mathematics/factorial-of-a-number-iterative.java
import java.io.*;
import java.util.*;

class FactorialOfANumberIterative {
    static int fact(int num) {
        int res = 1;

		for(int i=2; i<=num; i++) {
			res = res * i;
		}
		return res;
    }

    public static void main(String[] args) {
        System.out.println(fact(5));
    }
}

File: C#/Mathematics/factorial-of-a-number-iterative.cs
using System;

namespace FactorialOfANumberIterativeNameSpace {
    class FactorialOfANumberIterative {
        public int fact(int num) {
            int res = 1;

            for (int i=2; i<=num; i++) {
                res = res * i;
            }

            return res;
        }

        static void Main(string[] args) {
            FactorialOfANumberIterative factorialOfANumberIterative = new FactorialOfANumberIterative();
            Console.WriteLine(factorialOfANumberIterative.fact(5));
        }
    }
}